### PR TITLE
Added multiple props support

### PR DESCRIPTION
**Pull request by miketamis**
#1

### CODE REVIEW

**Review by iamchanii — COMMENTED** [review]
Good work. have you been tested these changes on your machine and does it work as you want to do? I haven't used Storybook with Relay recently, so I can't test it.

Let me know if it does.

File: src/decorators/withRelay.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export interface WithRelayParameters<
    * @param queryResult Result of the useLazyLoadQuery hook with the query passed as parameter.
    * @returns An entry to be added to the story's args.
    */
-  getReferenceEntry: (queryResult: TQuery['response']) => [string, unknown];
+  getReferenceEntry: (queryResult: TQuery['response']) => [string, unknown] || [string, unknown][];
```
Comment (iamchanii) [inline src/decorators/withRelay.tsx:33]: p3; How about change this into `getReferenceEntries`? I think it could be better for me.

Comment (miketamis) [reply src/decorators/withRelay.tsx:33]: ie have both a getReferenceEntry and getReferenceEntries? 

or make a breaking change?

Comment (iamchanii) [reply src/decorators/withRelay.tsx:33]: I think it's ok even we have both for now and throw an error if user provides both.

Comment (miketamis) [reply src/decorators/withRelay.tsx]: have made the change

**Comment by iamchanii** [comment]
> How do you build this project, if I use `tsc` it only create the declarations

I think I may have been running `yarn vite` manually.

**Comment by miketamis** [comment]
Tested it and it works, have formatted coded with dprint

**Pull request merged**